packages/adminrouter/extra/src/test-harness/tests/test_generic.config.yml

```diff
@@ -279,6 +279,44 @@ endpoint_tests:
       - master
       - agent
 
+######### /system/checks/ endpoint
+  - tests:
+      are_upstream_req_headers_ok:
+        skip_authcookie_filtering_test: true
+        auth_token_is_forwarded: skip
```

Comment:
Do we need to forward the DC/OS auth token to the upstream? 

I know it does not matter for Open (i.e. there is no auth token when communicating with agent), but this test will be also run for EE. So, can we please change it to `false` and add `proxy_set_header Authorization "";` to the added location blocks. If the upstream needs the token (i.e. it's an authorizer), let's please set it to `true`.

The idea is not to forward the auth token if this is not needed and to have it tested.

Thanks!